### ISSUE TITLE
Empty schema.org accessibility metadata not flagged as error

### ISSUE THREAD

**Issue by SarahMagdalena** [daisy/ace#207]
Please provide the following details if possible or relevant:

- [1.0.2] the version of Ace impacted by the issue (as returned by the `ace -v` command).

Currently, missing schema.org accessibility metadata (e.g. accessMode etc.) are indicated as an error, while empty metadata (e.g. `<meta property="schema:accessMode"></meta>`) are not. 
Is it possible to extend the rule to missing and/or empty metadata to make sure there is at least one description for every metadata property? Or is there a case, where empty schema.org accessibility metadata properties make sense?

**Comment by laudrain** [comment]
See also [Issue 202](https://github.com/daisy/ace/issues/202)
